.github/workflows/main.yml

```
@@ -55,11 +56,32 @@
         REPORTGENERATOR_LICENSE: ${{ secrets.REPORTGENERATOR_LICENSE }}
 
+    - name: Upload coverage history
+      uses: actions/upload-artifact@v4
+      if: success() && ${{ env.IS_TARGET_MAIN == 'true' }}
+      with:
+        name: CoverageHistory-${{ matrix.os }}
+        path: ./artifacts/TestResults/coveragehistory
+
+    - name: Upload coverage history
+      uses: actions/upload-artifact@v4
+      if: success() && ${{ env.IS_TARGET_MAIN == 'true' }}
```

actionlint (1.7.7)

if: condition "success() && ${{ env.IS_TARGET_MAIN == 'true' }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)